sdk/batch/Microsoft.Azure.Batch/tests/IntegrationTests/AddTaskCollectionIntegrationTests.cs

```diff
@@ -369,7 +369,7 @@ public async Task Bug1360227_AddTasksBatchWithFilesToStage(bool useJobOperations
                 Assert.Equal(expectedInitialFileStagingArtifactsCount, legArtifactsCountList.First());
                 Assert.Equal(expectedFinalFileStagingArtifactsCount, legArtifactsCountList.Last());
             },
-            TestTimeout);
+            LongTestTimeout);
```

Comment:
Don't forget to update the attribute to LongDuration, to reflect this change.

Reply:
Thanks, updated the Trait attribute to LongLongDuration as the average test time in my local env is ~5mins